templates/workspaces/base/terraform/azure-monitor/azure-monitor.tf

```diff
@@ -44,18 +44,41 @@ resource "azurerm_log_analytics_linked_storage_account" "workspace_storage_custo
   storage_account_ids   = [azurerm_storage_account.app_insights.id]
 }
 
-resource "azurerm_monitor_private_link_scope" "workspace" {
-  name                = "ampls-${var.tre_id}-ws-${local.short_workspace_id}"
-  resource_group_name = var.resource_group_name
-  tags                = var.tre_workspace_tags
+# TODO: Swith to azurerm once the issue is resolved
```

Comment:
Do you have a TF issue to link here?

Reply:
I don't, was a while ago and didn't get it written up, but know this resolved. 

@SvenAelterman  was the app insights one that you raised and tested this fix with me? Thanks.